From 1784be00b170308c56a3d467b22163a385985497 Mon Sep 17 00:00:00 2001
From: Tim Orling <ticotimo@gmail.com>
Date: Sun, 27 Feb 2022 15:28:18 -0800
Subject: [PATCH] lopper: fix wheel build

The wheel that is built is:
lopper-1.0-py3-none-any.whl

Set BASEVERSION to 1.0 and use this to set PV and the value for PYPA_WHEEL
(the path to the wheel filename).

Signed-off-by: Tim Orling <tim.orling@konsulko.com>
---
 recipes-kernel/lopper/lopper_git.bb | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/recipes-kernel/lopper/lopper_git.bb b/recipes-kernel/lopper/lopper_git.bb
index 606dab6..e96bac7 100644
--- a/recipes-kernel/lopper/lopper_git.bb
+++ b/recipes-kernel/lopper/lopper_git.bb
@@ -7,7 +7,10 @@ SRC_URI = "git://github.com/devicetree-org/lopper.git;branch=master;protocol=htt
 SRCREV = "3c81fcce08eeb64cbbde1535abd83572985a8689"
 S = "${WORKDIR}/git"
 
-PV="v1.0+git${SRCPV}"
+BASEVERSION = "1.0"
+PV="v${BASEVERSION}+git${SRCPV}"
+
+PYPA_WHEEL = "${PIP_INSTALL_DIST_PATH}/${BPN}-${BASEVERSION}-*.whl"
 
 LIC_FILES_CHKSUM = "file://LICENSE.md;md5=8e5f5f691f01c9fdfa7a7f2d535be619"
 
-- 
2.25.1

